### DISCUSSION TITLE
Make Threshold line visible on bar chart

### DISCUSSION THREAD

**Discussion by epraetor-eolian** [grafana/grafana#47673] [Dashboards, Panels & Visualizations]
Hi - is there a way to show the threshold as a horizontal line across the panel - similar to how it is on time series charts? Seems like it would be a useful feature to have.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
